docs/install/Knative-custom-install.md

```diff
@@ -152,6 +152,10 @@ service mesh. If you install any of the following options, you must install
    > Tip: You can append the `--watch` flag to the `kubectl get` commands to
    > view the pod status in realtime. You use `CTRL + C` to exit watch mode.
 
+## Installing Cert-Manager
```

Comment:
We should remove this section from all the installation guides because this indicates that "cert-manager" should be installed.  And I don't that it's possible at this point to install cert-manager especially since the requirement is to have a custom domain setup (that is a lot of steps and post install too).

I believe that installing and configuring cert-manager is an adv. topic that is performed some time after a successful knative installation.

Reply:
Deleted it from this file. Will delete this part from other files after we are good with change for the [comment](https://github.com/knative/docs/pull/1281#discussion_r283963989)